app/views/home/dashboard.html.erb

```diff
@@ -26,8 +26,8 @@
       </header>
       <div class="panel-body">
         <ul>
-          <% Project.order(created_at: :desc).take(5).each do |idea| %>
-            <li><%= link_to idea.name, idea %></li>
+          <% Project.joins(:project_votes).select('`projects`.id, `projects`.name, count(*) as count').group('`projects`.id').order('count DESC').take(5).each do |project| %>
+            <li><%= link_to project.name, project %></li>
           <% end %>
```

Comment:
Good call. I made scopes for both ideas and projects called top_voted, which takes in an optional number for the limit. I also changed the join to be a LEFT JOIN so that projects with no votes still have a chance of showing up on the list.